CHANGELOG.md

```diff
@@ -3,6 +3,26 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## [7.1.2] Preview Release
+### Added
+- Added support for JDK 11
```

Comment:
For all of these entries, you need to link the PR that we merged. Refer to what we did for 7.1.1 release.